yunwoo/programmers-1/행렬의 덧셈.js

```diff
@@ -0,0 +1,10 @@
+function solution(arr1, arr2) {
+  return arr1.map((v, i) => v.map((value, idx) => value + arr2[i][idx]));
+}
```

Comment:
```suggestion
function solution(arr1, arr2) {
  return arr1.map((innerArr, i) => innerArr.map((value, j) => value + arr2[i][j]));
}
```
- 2중 map을 이용한 풀이 잘봤습니다 👍 2중 map을 사용할 때도 2중 for문 때 처럼 `i`, `j`를 활용하시면 됩니다.  현재 풀이로는 i, idx가 의미가 중복되는 경향이 있어서 헷갈릴 수 있을 것 같습니다 😃
- 또한, v와 value도 마찬가지입니다 :) arr1에서 순회하는 것은 배열이기 때문에 내부의 배열이라는 의미의 `innerArr`로 변경하였습니다.
- 이렇게 혼동되는 변수명을 명확하게 짓는 것이 좋습니다 꼭 기억해주세요 !!

Reply:
- 헉 j가 생각이 안났네요.. 감사합니다
- innerArr 정말 적절한 이름이네요.. 저도 변수네임 명확하게 하도록 더 노력해보겠습니다!!!! 감사합니다👍